docs/release-notes/s3gw-v0.15.0.md

```diff
@@ -0,0 +1,39 @@
+# Release Notes - v0.15.0
+
+This release focuses on stabilizing our continuous integration and release process.
+In this context, we have also addressed a number of issues that was affecting our
+testing framework when automatically triggered by CI.
+
+This activity, although not introducing a direct improvement consumable by the user,
+is crucial to ensure a proper and stable environment for the upcoming major
+enhancements the s3gw's team is currently working on.
+
+Obviously, we still continued to address regular issues affecting all the s3gw's
```

Comment:
I'd get rid of "Obviously", perhaps something like "We continue to address the regular issues that affect all of s3gw's components."